rest/api_collections_test.go

```diff
@@ -194,6 +194,7 @@ func TestMultiCollectionDCP(t *testing.T) {
 		t.Skip("Test relies on import - needs xattrs")
 	}
 
+	t.Skip("Skip until CBG-2266 is implemented")
```

Comment:
Maybe stick a note on CBG-2266 about this test to ensure this doesn't get missed.